nemo/collections/asr/models/asr_model.py

```diff
@@ -53,7 +54,7 @@ def multi_test_epoch_end(self, outputs, dataloader_idx: int = 0):
         return {'test_loss': val_loss_mean, 'log': tensorboard_logs}
 
     @classmethod
-    def list_available_models(cls) -> 'List[PretrainedModelInfo]':
+    def list_available_models(cls) -> List[PretrainedModelInfo]:
         """
```

Comment:
Revert. We only use str quote for this to avoid circular dependency

Reply:
Reverted quotes